aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -1093,6 +1093,12 @@ where
         }
     }
 
+    fn materialize_agg_v1(
```

Comment:
Can we add a TODO, since we can implement this method and also compare against sequential execution results v. aggregator v1.